tools/downloader/list_topologies.yml

```diff
@@ -826,6 +854,7 @@ topologies:
 
   - name: "license-plate-recognition-barrier-0007"
     description: "TensorFlow LPRNet"
+    type: "ocr"
```

Comment:
I think that this should be called `optical_character_recognition`, for consistency.

Reply:
done